src/components/inputs/answers/SelectInput.jsx

```diff
@@ -0,0 +1,35 @@
+import { React } from 'react';
+
+const styles = `
+    .font-barlow {
+        font-family: 'Barlow', sans-serif;
+    }
+`;
+
+function SelectInput(props) {
+    console.log(props);
+    const options = props.input.options || [];
+    return (
+        <div className="rounded shadow bg-white font-barlow pt-3 px-0 mt-3">
+            <div className="d-flex column align-items-center justify-content-between m-0 pb-3">
```

Comment:
```suggestion
            <div className="row align-items-center justify-content-between m-0 pb-3">
```
- Como acomoda colunas, convém ser uma row dentro do sistema de grid do bs (d-flex e flex-column implícitos);